scripts/test.sh

```diff
@@ -0,0 +1,58 @@
+cwd=$(pwd)
+file_checker_root_path="$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )/.."
+cd $file_checker_root_path
+
+error_code=0
+
+echo '''
+================================================================================
+                                      Nose
+================================================================================
+'''
+nosetests . -v --with-coverage --cover-erase
+if [ $? -ne 0 ]; then
```

Comment:
Instead of checking `$?`, which could be error-prone, instead directly execute the command in the if statement.